docker-compose.yml

```diff
@@ -41,12 +41,13 @@ services:
   testing:
     depends_on:
       - app_db
-    image: wpgraphql-testing:latest
+    image: wp-graphql-testing:latest-wp${WP_VERSION-5.6}-php${PHP_VERSION-7.4}
     volumes:
       - '.:/var/www/html/wp-content/plugins/wp-graphql'
       - './.log/testing:/var/log/apache2'
       - './codeception.dist.yml:/var/www/html/wp-content/plugins/wp-graphql/codeception.yml'
-    env_file: .env.dist
+    env_file:
+      - .env
```

Comment:
@markkelnar so I do a lot of my testing locally outside of docker and change my .env file locally to credentials for my local MySQL environment, etc but don't touch my .env.dist file as that's used for testing. 

If we make Docker use the `.env` file, then we'll want to track changes to it, and I and others that run tests locally outside of Docker will have to be dilligent not to commit changes to the .env file. 

Why not keep `.env.dist` as the default, but use `.env` as an override locally if/when needed?

Reply:
Glad this got your attention. I saw your git history changes and had some questions. Let's chat.